typescript/sdk/src/providers/transactions/transformer/ethersV5/EV5InterchainAccountTxTransformer.ts

```diff
@@ -56,19 +65,23 @@ export class EV5InterchainAccountTxTransformer
       this.props.config,
     );
 
-    const transformedTxs: ethers.PopulatedTransaction[] = [];
-    for (const txChain of objKeys(txChainsToInnerCalls)) {
+    const transformedTxs: Promise<ethers.PopulatedTransaction>[] = [];
+    objMap(txChainsToInnerCalls, (destination, innerCalls) => {
       transformedTxs.push(
-        await interchainAccountApp.getCallRemote({
+        interchainAccountApp.getCallRemote({
           chain: this.props.chain,
-          destination: txChain,
-          innerCalls: txChainsToInnerCalls[txChain],
+          destination,
+          innerCalls,
           config: this.props.config,
           hookMetadata: this.props.hookMetadata,
         }),
       );
-    }
+    });
```

Comment:
nit: do not operate on unresolved promises unless necessary
why not do the concurrentMap here?

Reply:
i'm using objMap here since `txChainsToInnerCalls` is a record and i need to pull the `destination`

i ended up removing the unresolved promise logic (and concurrentMap altogether as result). should be greatly simplified now